docs/index.md

```diff
@@ -1,51 +1,47 @@
-<a href="https://discord.gg/C29hYvh"><img style="float: right;" src="img/discord.png"></a>
-
-# **NH Switch Guide**
-
-The NH Switch Guide is a collaboration from Nintendo Homebrew's Discord community, getting you from a stock Switch to Atmosphère.
-
-For live support with this guide, visit us in **#switch-assistance-1** or **#switch-assistance-2** over at the [NH-Discord server](https://discord.gg/C29hYvh).
+---
+layout: home
```

Comment:
Where are these layouts defined?

Reply:
It's built into VitePress. https://vitepress.dev/reference/default-theme-layout